src/index.js

```diff
@@ -153,6 +153,34 @@ async function handleDelete({ config, handler }, change, log) {
   }
 }
 
+/**
+ * Return a flag indicating whether a index record contains outdated data.
+ *
+ * @returns true if the record is outdated and shouldn't be used for indexing
+ */
+function isOutdated(record, headers, change) {
+  if (!change.uid || record.sourceHash === change.uid) {
+    // this is consistent
+    return false;
+  }
+  const lastModified = headers.get('last-modified');
+  if (!lastModified || !change.time) {
+    // unable to determine whether there is a discrepancy without dates
+    return false;
+  }
+  const lastModifiedMs = Date.parse(lastModified);
+  if (Number.isNaN(lastModifiedMs)) {
+    // last modified date is unusable
+    return false;
+  }
+  const eventTimeMs = Date.parse(change.time);
+  if (Number.isNaN(eventTimeMs)) {
+    // event time is unusable
+    return false;
+  }
+  return eventTimeMs > lastModifiedMs;
```

Comment:
so, if the event is newer than the last modifed, it is outdated ???

Reply:
Not quite: if the UIDs (or source hashes) are different, and the UID at the event time is different than the UID of the HTML found, and the HTML is older, **then** it is outdated. I'll add that to the documentation of the function.